deformable_manipulation_experiment_params/include/deformable_manipulation_experiment_params/ros_params.hpp

```diff
@@ -873,6 +912,12 @@ namespace smmap
         return val.GetImmutable();
     }
 
+    // diminishing with Mengyao's controller
+    inline bool GetUseDiminishingModelWithSamplingController(ros::NodeHandle& nh)
```

Comment:
Is this function (and parameter name) still accurate? Or does this trigger using the nomad optimization controller?